src/current/v25.1/recommended-production-settings.md

```diff
@@ -22,6 +22,15 @@ For optimal cluster performance, Cockroach Labs recommends that all nodes use th
 
 We recommend running a [glibc](https://www.gnu.org/software/libc/)-based Linux distribution and Linux kernel version from the last 5 years, such as [Ubuntu](https://ubuntu.com/), [Red Hat Enterprise Linux (RHEL)](https://www.redhat.com/technologies/linux-platforms/enterprise-linux), [CentOS](https://www.centos.org/), or [Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs).
 
+New deployments should configure [transparent huge pages (THP)](https://www.kernel.org/doc/html/latest/admin-guide/mm/transhuge.html) with the `madvise` option:
```

Comment:
Maybe give a bit of the reasoning: "We have observed (in rare cases) increased memory usage due to transparent huge pages (THP) being enabled (set to `always`). New deployments should ..."

Reply:
added, PTAL!